app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -2,8 +2,6 @@ module Hyrax
   class Admin::AdminSetsController < ApplicationController
     include Hyrax::CollectionsControllerBehavior
 
-    before_action :ensure_manager!, except: [:show]
-    before_action :ensure_viewer!, only: [:show]
     load_and_authorize_resource
```

Comment:
See the explanation for why these are being removed in the main description of the PR.